### PR TITLE
Port from 18.3 - add protection for downloading bitstream and DR acce…

### PR DESCRIPTION
**Pull request by houlz0507**
…sses

### CODE REVIEW

File: src/runtime_src/driver/xclng/tools/xbutil/xbutil.h [hunk 1]
```diff
@@ -959,20 +963,21 @@ class device {
...
-                  cu_ba = sensor_tree::pretty<int>(subv.second.get_value<int>(), "N/A", true);
-                else if( subv.first == "status" )
+                else if( subv.first == "base_address" ) {
+                  auto addr = subv.second.get_value<uint64_t>();
+		  cu_ba = (addr == (uint64_t)-1) ? "N/A" : sensor_tree::pretty<uint64_t>(addr, "N/A", true);
+		} else if( subv.first == "status" )
                   cu_s = subv.second.get_value<std::string>();
               }
-              ostr << "CU[" << std::right << std::setw(2) << cu_i << "]: "
+              ostr << "CU[" << std::right << std::setw(2) << cu_i++ << "]: "
```
Comment (keryell) [inline src/runtime_src/driver/xclng/tools/xbutil/xbutil.h]: Having this `cu_i++` lost in the middle of an output statement makes the code difficult to its own line?

Comment (houlz0507) [reply src/runtime_src/driver/xclng/tools/xbutil/xbutil.h]: did this. Thanks for your comment.

**Comment by stsoe** [comment]
retest this please

**Comment by houlz0507** [comment]
retest this please

**Pull request merged**